backend-shared/config/db/db_field_constants.go

```diff
@@ -175,6 +175,6 @@ var DbFieldMap = map[string]int{
 }
 
 // Get value of constants based on constant variable name given as String.
-func getConstantValue(variable string) int {
+func GetConstantValue(variable string) int {
```

Comment:
As @drpaneas mentioned, this can be kept at `getConstantValue`, and the tests can be kept in the package they were defined in (e.g. `db`, rather than `db_test` package... unless there was a good reason to move them to a new package?)